rust-runtime/aws-smithy-http-server/src/plugin/check_health.rs

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Middleware for handling health check requests.
```

Comment:
Little example in the docs here perhaps?

Reply:
Done in https://github.com/awslabs/smithy-rs/pull/2540/commits/5bed399c97fb322d9e779b0576cb660d8deef0f8.